lib.php

```diff
@@ -81,52 +84,12 @@ public static function has_config() {
 
     public static function get_config_options() {
 
-        // Get default config;
-        $defaultconfig = get_objectfs_config();
+      $defaultconfig = self::get_default_config();
```

Comment:
Dint know what happened to your indentation here - should be running phpCS to have consistent formatting.